dynamics/tests/FakeSmeshData.cpp

```diff
@@ -0,0 +1,192 @@
+/*!
+ * @file FakeSmeshData.cpp
+ *
+ * @date Dec 14, 2023
+ * @author Tim Spain <timothy.spain@nersc.no>
+ */
```

Comment:
What is this for? Documentation would be useful

Reply:
A testing data source. I have added a brief description to each of the relevant files.

Reply:
Where?

Reply:
Ah, found it.